packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "32.0.0",
+  "version": "32.0.1",
```

Comment:
Hi @tjheffner thank you for this update! Do you mind reverting this `core` version change? The design system team will handle updating this when a new component library release is made.

Reply:
Not at all! There were some notes in the PR template so I assumed I had to do it for it to go live. Will revert.